Cargo.toml

```diff
@@ -331,6 +331,7 @@ omicron-certificates = { path = "certificates" }
 omicron-passwords = { path = "passwords" }
 omicron-workspace-hack = "0.1.0"
 oxlog = { path = "dev-tools/oxlog" }
+oxnet = { git = "https://github.com/oxidecomputer/oxnet", branch = "main" }
```

Comment:
Are we relying on the lockfile to keep this in sync? I usually don't like relying on a branch directly. Can we give it a 1.0 version tag or something?

Reply:
For now; I want to get to something resembling stable in oxnet, publish to crates.io and depend on that. We have several other branch = main deps

Reply:
Cool, works for me.